app/models.py

```diff
@@ -1091,6 +1091,131 @@ def serialize(self):
         return self.serializable
 
 
+class UserView(db.Model):
```

Comment:
It will be there as well as the User object, most of the code is copied. But there is no way around this, shorting of changing the ORM entirely.

I'm really unhappy with the way this has to be set up, but there are several situations now where being able to use calculated values in views is important.